### DISCUSSION TITLE
Change close button

### DISCUSSION THREAD

**Discussion by el-klo** [pboivin/filament-peek#136] [Q&A] [answered]
I can't seem to find where to edit the preview modal close button. I want to change the language. My whole app is in Dutch and only this button keeps displaying "close".

**Comment by pboivin** [comment]
Hi @el-klo, I'd love if you could help me translate the plugin to Dutch! If you have some time, could you send me a PR? or just copy-paste here the translated version of this file : 
https://github.com/pboivin/filament-peek/blob/2.x/resources/lang/en/ui.php

**Comment by el-klo** [comment, marked as answer]
Sure thing! Here you go. I kept preview just as preview, it could be translated to "voorvertoning" but i think most Dutch people just use preview. If you need anymore, just let me know. 

```
<?php

return [
    'preview-action-label' => 'Preview',

    'preview-modal-title' => 'Preview',

    'close-modal-action-label' => 'Sluiten',

    'builder-editor-title' => 'Editor',

    'refresh-action-label' => 'Preview verversen',

    'editor-settings-label' => 'Instellingen',

    'editor-auto-refresh-label' => 'Automatisch verversen',

    'accept-action-label' => 'Accepteren',

    'discard-action-label' => 'Weggooien',
];
```

**Comment by pboivin** [comment]
@el-klo Thank you! I just tagged v2.2.8 with your contribution : 
https://github.com/pboivin/filament-peek/releases/tag/v2.2.8
